doc/api/esm.md

```diff
@@ -1328,11 +1336,7 @@ If the code needs more advanced `require` features, it will have to construct
 its own `require` using  `module.createRequire()`.
 
 ```js
-/**
- * @returns {string} Code to run before application startup
- */
-export function getGlobalPreloadCode() {
-  return `\
+export const globalPreloadCode = `\
```

Comment:
We were thinking of adding a channel to the preload code. By removing the function wrapping, that seems harder. It also means that the loader cannot return different preload code for different contexts/threads.

Reply:
I removed the function wrapping to simplify how evaluation of multiple loaders interleaves before user code runs. I think it is likely this will change again when I work on threading.

Reply:
Can we reduce the churn then and not change it here? There is a PR for threading and pre-existing discussion on how preload code would interact with threading. Removing the function wrapping seems weird if we expect that the same loader hooks may be called from different contexts.

Reply:
I suppose i can, my goal was to avoid reentry into the loaders during bootstrapping, which is why changing it back to a function would be fine once they're threaded.